assembly/src/ast/imports.rs

```diff
@@ -29,7 +31,11 @@ impl ModuleImports {
     /// # Panics
     /// Panics if the number of imports is greater than MAX_IMPORTS, or if the number of invoked
     /// procedures is greater than MAX_INVOKED_IMPORTED_PROCS
-    pub fn new(imports: ImportedModulesMap, invoked_procs: InvokedProcsMap) -> Self {
+    pub fn new(
+        imports: ImportedModulesMap,
+        invoked_procs: InvokedProcsMap,
+        reexported_procs_paths: Vec<LibraryPath>,
```

Comment:
Should this be a `BTreeSet`?

Also, this contains module paths rather than procedure paths, right? If so, I'd call it `reexported_modules`.